README.md

```diff
@@ -187,133 +218,60 @@ You can deploy the devdocs site locally using [this Vagrant project](https://git
 
 If you have questions, open an issue and ask us. We're looking forward to hearing from you!
 
-*	<a href="https://twitter.com/MagentoDevDocs" class="twitter-follow-button" data-show-count="false">Follow @MagentoDevDocs</a>
-
-*	<a href="mailto:DL-Magento-Doc-Feedback@magento.com">E-mail us</a>
-
-*	<a href="https://devdocs.magento.com">Visit our documentation site</a>, built using [GitHub pages](https://pages.github.com/).
+- [Follow @MagentoDevDocs](https://twitter.com/MagentoDevDocs)
+- [E-mail us](mailto:DL-Magento-Doc-Feedback@magento.com)
+- [Visit our documentation site](https://devdocs.magento.com), built using [GitHub pages](https://pages.github.com/).
 
-## Build DevDocs in Windows
+## Build on Windows
 
-Some of the technologies we use to develop DevDocs is not compatible with Windows, such as [Jekyll](https://jekyllrb.com/docs/windows/). For this reason, we do not support DevDocs management in Windows; however, we have documented the following procedures to build the DevDocs in a Windows environment. Any further use of this setup or troubleshooting is up to you. 
+Windows does not provide the out-of-the-box Ruby, so we recommend to use a virtual environment with Docker or Vagrant described above.
```

Comment:
Do we? I don't know. I had my Cloud windows users look at this before and I disagree. I did a lot of research into the Windows method I documented.